src/components/marketplace/order_book.tsx

```diff
@@ -36,7 +36,7 @@ const orderToRow = (
     selectedToken: Token,
     mySizeOrders: OrderBookItem[] = [],
 ) => {
-    const size = tokenAmountInUnits(order.size, selectedToken.decimals);
+    const size = tokenAmountInUnits(order.size, selectedToken.decimals, 7);
```

Comment:
I think we should decide how may decimals the UI is going to display (and hide, implicitly) and create the corresponding constants to replace the magic numbers in the functions `padRightSplitted` and `tokenAmountInUnits`.

Based on "Main Market page" mockup, the Order Book may show:
* 4 decimals for "ORDER SIZE" and "MY SIZE" columns.
* 7 decimals for "PRICE (ETH)" column.

Reply:
Also, just in case: apply the fix to "MY SIZE" column's values.

Reply:
It's done

Reply:
it's done